Dockerfile

```diff
@@ -66,6 +66,7 @@ RUN useradd --create-home --shell /bin/bash --uid 900 --gid 900 lbdumps
 
 RUN groupadd --gid 901 listenbrainz_stats_cron
 RUN useradd --create-home --shell /bin/bash --uid 901 --gid 901 listenbrainz_stats_cron
+RUN mkdir /logs && chown lbdumps:lbdumps /logs
```

Comment:
This will need a matching `--volume` flag in docker-server-configs. is that on your list?

Reply:
yep, I have a branch for that as well.

Reply:
In fact that PR is merged already: https://github.com/metabrainz/docker-server-configs/blob/master/scripts/services.sh#L1520